assets/js/components/Config/PropertyCollapsible.vue

```diff
@@ -6,8 +6,8 @@
 			type="button"
 			@click="toggle"
 		>
-			<span v-if="open">Hide advanced settings</span>
-			<span v-else>Show advanced settings</span>
+			<span v-if="open">{{ $t("config.mqtt.hideAdvancedSettings") }}</span>
+			<span v-else>{{ $t("config.mqtt.showAdvancedSettings") }}</span>
```

Comment:
```suggestion
			<span v-else>{{ $t("config.general.showAdvancedSettings") }}</span>
```

This is a general component that is also used in other dialogues.

Reply:
Great, will update this accordingly